### PR TITLE
Feature/10 print events daily view

### LINKED ISSUE #10: Print events (daily view)
Create a readable format and layout for the daily events. This should be printed out to the worker containing *all* the information.
(A worker should only get the information for their jobs for that day)
EST 5

### PR DESCRIPTION
**Pull request by szdariusz**
Preview and print functionality for events assigned to a specific user.

### CODE REVIEW

File: wman-frontend/src/components/Print-functionality/Print-events/print-events.component.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            .then(response => setCalendarEvents(response.data))
+            .catch(error => console.log(error))
+    }, [axios]);
+
+   const handleChange = (e) => {
+        setSearched(e.target.value);       
+    }
+    
+    const filteredEvents = calendarEvents.filter(        
+        event => event.assignedUsers[0].username.includes(searched)
```
Comment (tsaskater) [inline wman-frontend/src/components/Print-functionality/Print-events/print-events.component.jsx]: This only checks for the first element of the assigned users which means the printed document won't be present when choosing a secondly assigned user or anything beyond the first. pls fix 🎅🏻🤹🏿‍♂️

**Review by tsaskater — CHANGES_REQUESTED** [review]

**Review by tsaskater — APPROVED** [review]
Works really good! Good Job!
I'm gonna approve it but pls commit one more thing.
When a worker has no assigned jobs please display it somehow, same with errors. At the moment it's just empty, there is no visual feedback. (Empty=No jobs for the day? error?-> I go the console/network, an average user will not get that far) 
IMO this improves the user experience, also it's easier for us to understand why there are no printable events.

**Comment by szdariusz** [comment]
I did it

**Review by tamas-lengyel — APPROVED** [review]
Looks good, nice job!

**Review by pop-simon — APPROVED** [review]
It works well, good job! 👏

**Pull request merged**